protected/humhub/modules/ui/menu/MenuEntry.php

```diff
@@ -117,6 +117,9 @@ public function setIsActiveState($moduleId, $controllerIds = [], $actionIds = []
         return $this;
     }
 
+    /**
+     * @deprecated since 1.17, use humhub\helpers\ControllerHelper::isActivePath()
+     */
```

Comment:
Can we use the new helper method here directly and delete the old code?

Reply:
https://github.com/humhub/humhub/pull/7188/commits/636702fc63f70a8dac7b0d0f1a31164c70e82f34#diff-f75bf5902a11d3cf3a7a3055ff73056cf76bbd7c6aa94849bff70d287a01f6eaR126